doc/revision_history.rst

```diff
@@ -6,6 +6,18 @@
 Revision History
 ################
 
+v5.4.0
+======
+
+Major changes:
+
+* Add support for the new ``setGroupId`` ``Script``  command and ``groupId`` Script event to `BaseScript`:
```

Comment:
Should the second `Script` also have double backticks around it?

Reply:
Yes, and I ended up moving groupId to the Script state event, so I need to fix this text.